lib/constants.js

```diff
@@ -1053,7 +1053,7 @@ export const DEFAULT_SETTINGS = {
   watermark: false,
   squaredImage: false,
   hiddenCharacters: false,
-  preset: 'preset:4'
+  id: 'preset:4'
```

Comment:
I believe you want to just remove this key altogether

Reply:
You're right, I added another commit to remove the key from default :)